install.rst

```diff
@@ -63,7 +63,7 @@ checksum then run for example the following command: ::
 Virtualization platform-specific notes:
 
 - For VMWare, configure the virtual machine to use the IDE driver for the disk.
-- For Proxmox, change the default CPU model to anything other than ``kvm64``.
+- For Proxmox, for maximum performance, select ``host`` for the CPU type. Avoid "kvm64", refer to Proxmox documentation for further details.
```

Comment:
```suggestion
- For Proxmox, for maximum performance, select ``host`` for the CPU type. Avoid "kvm64", because Rocky Linux image does not support it.  Refer to `Proxmox documentation <https://pve.proxmox.com/pve-docs/chapter-qm.html#qm_cpu>`_ for further details about CPU selection.
```